presto-password-authenticators/src/main/java/io/prestosql/plugin/password/file/FileBasedAuthenticationConfig.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.password.file;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.concurrent.TimeUnit;
+
+public class FileBasedAuthenticationConfig
+{
+    private String configFile;
+    private Duration refreshInterval = new Duration(10, TimeUnit.MILLISECONDS);
```

Comment:
This seems too low. Maybe make it 5 or 10 seconds?

Reply:
`private Duration refreshInterval = new Duration(5, TimeUnit.SECONDS);` : Done